package.json

```diff
@@ -135,5 +135,6 @@
     },
     "dependencies": {
         "highcharts": "^12.1.2"
-    }
+    },
+    "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
I removed this from the bar chart pr with an extra commit - I think you'll need to do the same here.